sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
+approach delivers better performance compared to individual networks. They
+compared a number of feed-forward architectures trained on RNA-Seq data
+with and without an additional set of CLIP-seq, knockdown, and over-expression
+based input features. Results showed that the integrative deep model
+generalized well for combined data, offering large performance improvement
+for alternative splicing event estimation. Chen et al.
+[@tag:Chen2015_trans_species] used deep belief networks to learn
+phosphorylation states of a common set of signaling proteins in primary
+cultured bronchial cells collected from rats and humans treated
+with distinct stimuli. Interpreting species as different modalities
+representing similar high-level concepts, they showed that DBNs were able
+to capture cross-species representation of signaling mechanisms in response
+to a common stimuli. Another application used DBNs for joint unsupervised
+feature learning from cancer datasets contained gene expression,
+DNA methylation, and miRNA expression data [@tag:Liang2015_exprs_cancer].
+This approach allowed for the capture of intrinsic relationships in
+different modalities and for better clustering performance over conventional
+k-means based methods.
+
+Deep graphical models such as DBNs are considered to be well suited for
+multimodal learning tasks since they are aimed to learn joint probability
+distribution from inputs. However, it is difficult task that requires
+specialized algorithms for training and sampling.  While it is possible
+to fine-tune these models for discriminatory tasks, they are typically
+outperformed by commonly used convolutional neural networks (CNN)
+trained with backpropagation. Multimodal learning with CNNs is usually
+implemented as a collection of individual networks, in which each learns
+representations from single data type. These individual representations are
+further concatenated before or within fully-connected layers. FIDDLE
+[@tag:Eser2016_fiddle] is an example of multimodal CNNs that represents an
+ensemble of individual networks that take as inputs a number of genomic
+datasets, including NET-seq, MNase-seq, ChIP-seq, RNA-seq, and
+raw DNA sequence to predict Transcription Start Site-seq (TSS-seq) outputs.
+The combined model radically improves performance over separately trained
+datatype-specific networks, suggesting that it learns the synergistic
+relationship between datasets.
+
+Despite demonstrated improvements, transfer learning approaches also pose
+a number of challenges. As mentioned above, there are no theoretically sound
+principles for pre-training and fine-tuning. Most best practice recommendations
+are heuristic and have to take into account additional hyper-parameters that
+depend on specific deep architectures, sizes of pre-training and target
+datasets, and similarity of domains. Unfortunately, not many negative results
+are presented in study publications. As rare example, DeepChem, a framework
```

Comment:
DeepChem discussion [shortened](https://github.com/greenelab/deep-review/pull/347/files#diff-2f7eae35d8b5136fc03e28b1f36ab9f4R385)

Also [mentioned](https://github.com/greenelab/deep-review/pull/347/files#diff-2f7eae35d8b5136fc03e28b1f36ab9f4R369) limitations on assessing "similarity"

Feel free to add more, esp for related to chemical screening.